src/context/videoContext.js

```diff
@@ -99,12 +101,16 @@ const [ videoState, videoDispatch ] = useReducer(videoReducerFun, initialState);
     },[]);
 
     const addItemToLikedVideos = async (token,video) =>{
-        try{
-            console.log("entered like page");
-            const response = await addItemToLikedVideosHandler(token,video);
-            videoDispatch({type: "SET_LIKED_VIDEOS", payload: response.likes})
-        }catch(error){
-            console.log(error);
+        if(token){
+            try{
+                console.log("entered like page");
+                const response = await addItemToLikedVideosHandler(token,video);
+                videoDispatch({type: "SET_LIKED_VIDEOS", payload: response.likes})
+            }catch(error){
+                console.log(error);
```

Comment:
You can add a toast here to show the error message to the user.

Reply:
Okay